Snakefile

```diff
@@ -696,9 +782,37 @@ rule tokenize_parallel_opensubtitles:
     output:
         "data/tokenized/opensubtitles/parallel/{langpair}.{lang}.txt"
     shell:
-        "xc tokenize -l {wildcards.lang} {input} {output}"
+        "xc tokenize -f -l {wildcards.lang} {input} {output}"
+
+rule tokenize_paracrawl:
+    # Tokenize the text from Paracrawl, in one language at a time, but keeping
+    # track of the language pair they originated from.
+    input:
+        "data/extracted/paracrawl/{langpair}.{lang}.txt"
+    output:
+        "data/tokenized/paracrawl-paired/{langpair}.{lang}.txt"
+    shell:
+        "xc tokenize -f -l {wildcards.lang} {input} {output}"
+
```

Comment:
When I tried to run `tokenize` it failed because I didn't have `ordered_set` in my virtualenv. I don't think it's listed in the requirements.